README.md

```diff
@@ -77,36 +79,29 @@ Parameters are provided to configure the behavior of the bridge. These parameter
  * (ROS 2) __num_threads__: The number of threads to use for the ROS node executor. This controls the number of subscriptions that can be processed in parallel. 0 means one thread per CPU core. Defaults to `0`.
  * (ROS 2) __max_qos_depth__: Maximum depth used for the QoS profile of subscriptions. Defaults to `10`.
 
+## Building from source
 
-## Clients
-
-[Foxglove Studio](https://foxglove.dev/studio) connects to `foxglove_bridge` for live robotics visualization.
-
-### Building from source
-
-You can also try `foxglove_bridge` by building from source.
-
-#### Fetch source and install dependencies
+### Fetch source and install dependencies
 
 ```bash
-cd <path/to/your/ros_ws>
+mkdir -p workspace/src
```

Comment:
I think we should leave the `cd <path/to/your/ros_ws>` line to give context that the following commands should be run inside your ROS workspace

Reply:
```suggestion
cd <path/to/your/ros_ws>
```